doc/radosgw/s3/commons.rst

```diff
@@ -20,9 +20,20 @@ The second method identifies the bucket via a virtual bucket host name. For exam
 
 To configure virtual hosted buckets, you can either set ``rgw_dns_name = cname.domain.com`` in ceph.conf, or add ``cname.domain.com`` to the list of ``hostnames`` in your zonegroup configuration. See `Ceph Object Gateway - Multisite Configuration`_ for more on zonegroups.
 
+.. prompt:: bash $
+
+	ceph config set client.rgw.<ceph authx client for rgw> rgw rgw_dns_name cname.domain.com
+
 .. tip:: We prefer the first method, because the second method requires expensive domain certification and DNS wild cards.
 
-.. tip:: You can define multiple hostname directly with the :confval:`rgw_dns_name` parameter.
+.. tip:: You can define multiple hostnames directly with the :confval:`rgw_dns_name` parameter.
+
+.. tip:: When SSL is enabled, the certificates need to use a wildcard in the domain name so it matches the bucket subdomains.
+
+.. tip:: RGW instances would share a dns name if they're behind the same proxy.Then you can use `ceph config set client.rgw` to set the dns name for all instances. 
+
+::
+	If the `s3website` api defined, then use `rgw_dns_s3website_name` need to set instead of `rgw_dns_name`.
```

Comment:
> .. note:: The static website view for the \`s3website\` api must be served under a different domain name. This is configured separately from \`rgw_dns_name\` in \`rgw_dns_s3website_name\`.

Reply:
API is enabled, then  `rgw...` needs to be set